doc/users/credits.rst

```diff
@@ -333,136 +575,272 @@ Simon Gibbons,
+The Gitter Badger,
 Thomas A Caswell,
 Thomas Hisch,
 Thomas Kluyver,
 Thomas Lake,
+Thomas Mansencal,
 Thomas Robitaille,
 Thomas Spura,
+Thomas VINCENT,
+Thorsten Liebig,
+Tian Xia,
 Till Stensitzki,
+Tim Hoffmann,
 Timo Vanwynsberghe,
+Tobias Froehlich,
 Tobias Hoppe,
 Tobias Megies,
 Todd Jennings,
 Todd Miller,
+Tom,
+Tom Augspurger,
+Tom Dupré la Tour,
 Tomas Kazmar,
 Tony S Yu,
 Tor Colvin,
 Travis Oliphant,
 Trevor Bekolay,
+Trish Gillett-Kawamoto,
+Truong Pham,
+Tuan,
+Tuan Dung Tran,
+Tuan333,
 Ulrich Dobramysl,
 Umair Idris,
+V. R,
 Vadim Markovtsev,
 Valentin Haenel,
+Valentin Schmidt,
+Vedant Nanda,
 Victor Zabalza,
+Vidur Satija,
 Viktor Kerkez,
 Vlad Seghete,
 Víctor Terrón,
 Víctor Zabalza,
+WANG Aiyong,
+Warren Weckesser,
 Wen Li,
 Wendell Smith,
 Werner F Bruhin,
 Wes Campaigne,
 Wieland Hoffmann,
+Will Silva,
+William Granados,
+William Mallard,
 William Manley,
 Wouter Overmeire,
 Xiaowen Tang,
+Xufeng Wang,
 Yann Tambouret,
+Yao-Yuan Mao,
 Yaron de Leeuw,
 Yu Feng,
 Yunfei Yang,
 Yuri D'Elia,
 Yuval Langer,
+ZWL,
 Zac Hatfield-Dodds,
+Zac-HD,
 Zach Pincus,
 Zair Mubashar,
+Zbigniew Jędrzejewski-Szmek,
+Zohreh,
+ahed87,
+akrherz,
+alcinos,
 alex,
+alvarosg,
+aneda,
 anykraus,
+apodemus,
 arokem,
+as691454,
 aseagram,
+ash13,
 aszilagyi,
 bblay,
+bduick,
 bev-a-tron,
 blackw1ng,
 blah blah,
+buefox,
 burrbull,
 butterw,
 cammil,
 captainwhippet,
+castany,
+cclauss,
+cgohlke,
+ch3rn0v,
 chadawagner,
 chebee7i,
+chelseatroy,
+cknd,
+cld,
+cldssty,
+clintval,
 danielballan,
 davidovitch,
 daydreamt,
+deeenes,
+deepyaman,
+dlmccaffrey,
 domspad,
 donald,
 drevicko,
 e-q,
 elpres,
 endolith,
+et2010,
 fardal,
 ffteja,
 fgb,
 fibersnet,
+fredrik-1,
 frenchwr,
+fuzzythecat,
 fvgoto,
+gcallah,
+gepcel,
 gitj,
 gluap,
+gnaggnoyil,
 goir,
+goldstarwebs,
+greg-roper,
+gregorybchris,
+hannah,
+helmiriawan,
 hugadams,
+ilivni,
 insertroar,
 itziakos,
+jacob-on-github,
 jbbrokaw,
+jbhopkins,
+jerrylui803,
+jhelie,
+jli,
+jonchar,
 juan.gonzalez,
+kalagau,
 kcrisman,
+keithbriggs,
 kelsiegr,
 khyox,
 kikocorreoso,
+klaus,
+klonuo,
 kramer65,
 kshramt,
+lboogaard,
+legitz7,
 lichri12,
 limtaesu,
+lspvic,
+luz.paz,
+lzkelley,
+mamrehn,
 marky,
 masamson,
+mattip,
 mbyt,
 mcelrath,
+mcquin,
 mdipierro,
+mitch,
+mlub,
+mobando,
 mrkrd,
+muahah,
+myyc,
+navdeep rana,
+nbrunett,
+nemanja,
+neok-m4700,
+nepix32,
 nickystringer,
+nmartensen,
 nwin,
+ob,
+patniharshit,
+pdubcali,
 pkienzle,
+productivememberofsociety666,
 profholzer,
 pupssman,
 rahiel,
+rebot,
 rhoef,
+rishikksh20,
+roonjoot,
 rsnape,
+ruin,
+rvhbooth,
+s0vereign,
 s9w,
+scls19fr,
+scott-vsi,
 sdementen,
+serv-inc,
+settheory,
 sfroid,
+shaunwbell,
+simonpf,
+sindunuragarp,
 sohero,
 spiessbuerger,
 stahlous,
+stone,
+stonebig,
 switham,
 syngron,
+thuvejan,
+tmdavison,
+tomoemon,
+tonyyli,
 torfbolt,
 u55,
 ugurthemaster,
+ultra-andy,
+vab9,
 vbr,
+vidursatija,
+vraelvrangr,
+watkinrt,
 xbtsw,
-and xuanyuansen.
+xuanyuansen,
+yinleon,
+zhangeugenia,
+zhoubecky,
+Élie Gouzien
```

Comment:
Something based on locale.strcoll should sort this properly at "E".

Reply:
`locale.strcoll` would need a `sorted(cmp=)` which was recently removed. Tried with `contributors.sort(key=locale.strxfrm)` but that does not work for some reason.